eipw-lint-js/src/lib.rs

```diff
@@ -23,6 +23,138 @@ use std::pin::Pin;
 
 use wasm_bindgen::prelude::*;
 
+#[wasm_bindgen(typescript_custom_section)]
+const TS_APPEND_CONTENT: &'static str = r#"
+ export type Author<S> = S;
+ 
+ export type Date<S> = S;
+ 
+ export type DefaultLint<S> =
```

Comment:
Can you think of a way we can get a compile or test failure if this list gets out of sync with [`DefaultLint`](https://github.com/ethereum/eipw/blob/411b101950ace4fa03f2da8ba8912d9111a0c0c2/eipw-lint/src/lints/known_lints.rs#L18)?